src/main/java/net/mcreator/element/converter/fv35/TooltypeProcedureConverter.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2022, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.converter.fv35;
+
+import com.google.gson.JsonElement;
+import net.mcreator.element.GeneratableElement;
+import net.mcreator.element.converter.IConverter;
+import net.mcreator.element.types.Procedure;
+import net.mcreator.util.BlocklyHelper;
+import net.mcreator.util.XMLUtil;
+import net.mcreator.workspace.Workspace;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.StringReader;
+import java.io.StringWriter;
+
+public class TooltypeProcedureConverter implements IConverter {
+	private static final Logger LOG = LogManager.getLogger("TooltypeProcedureConverter");
+
+	@Override
+	public GeneratableElement convert(Workspace workspace, GeneratableElement input, JsonElement jsonElementInput) {
+		Procedure procedure = (Procedure) input;
+		try {
+			procedure.procedurexml = fixXML(procedure.procedurexml);
+		} catch (Exception e) {
+			LOG.warn("Failed to convert replaced procedure " + input.getModElement().getName());
+		}
+		return procedure;
+	}
+
+	@Override public int getVersionConvertingTo() {
+		return 35;
+	}
+
+	protected String fixXML(String xml) throws Exception {
+		DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+		DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
+		Document doc = dBuilder.parse(new InputSource(new StringReader(xml)));
+		doc.getDocumentElement().normalize();
+		net.mcreator.util.BlocklyHelper bh = new net.mcreator.util.BlocklyHelper(doc);

```

Comment:
The BlocklyHelper is unused and can be removed, as well as its import

Reply:
oh yeah, I made a new block for the item at first when I realised that I don't have to, and forgot to remove that.